nimbus.fml.yaml

```diff
@@ -508,6 +508,22 @@ features:
             "move-actions": true,
             "toolbar-changes": true
         }
+  creditcard-autofill:
+    description: This property defines whether credit card autofill is enabled / disabled
```

Comment:
I would argue that the top level feature shouldn't be the on/off status, that's the variable. The top level nimbus feature should always be forward looking. So something like `This property defines the credit card autofill feature`